Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
+
+# Install dependencies.
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+     python python3-pip python3-tk texlive-latex-base latexmk git emacs locales
+
+# Configure UTF-8 encoding.
+RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && locale-gen
+ENV LANG en_US.UTF-8  
+ENV LANGUAGE en_US:en  
+ENV LC_ALL en_US.UTF-8 
+
+
+# Get a current copy of Cirq from Github.
+RUN git clone https://github.com/quantumlib/Cirq
```

Comment:
Oh, this is a docker file for the dev environment? Is that what people normally expect the dockerfile to install?

Reply:
The main goal is for dev environment, although it can be used for deployment too. The goal was to create an easy way to spin up a dev environment, without messing with host settings.